src/_P043_ClkOutput.ino

```diff
@@ -53,25 +55,20 @@ boolean Plugin_043(byte function, struct EventStruct *event, String& string)
 
     case PLUGIN_WEBFORM_LOAD:
       {
-        String options[3];
-        options[0] = "";
-        options[1] = F("Off");
-        options[2] = F("On");
-
+ 
         for (byte x = 0; x < PLUGIN_043_MAX_SETTINGS; x++)
         {
         	addFormTextBox(String(F("Day,Time ")) + (x + 1), String(F("p043_clock")) + (x), timeLong2String(ExtraTaskSettings.TaskDevicePluginConfigLong[x]), 32);
-//          addHtml(F("<TR><TD>Day,Time "));
-//          addHtml(x+1);
-//          addHtml(F(":<TD><input type='text' name='plugin_043_clock"));
-//          addHtml(x);
-//          addHtml(F("' value='"));
-//          addHtml(timeLong2String(ExtraTaskSettings.TaskDevicePluginConfigLong[x]));
-//          addHtml("'>");
-
-          addHtml(" ");
-          byte choice = ExtraTaskSettings.TaskDevicePluginConfig[x];
-          addSelector(String(F("p043_state")) + (x), 3, options, NULL, NULL, choice);
+          if (CONFIG_PIN1 >= 0) {
+            String options[3];
+            options[0] = "";
+            options[1] = F("Off");
+            options[2] = F("On");
+            addHtml(" ");
+            byte choice = ExtraTaskSettings.TaskDevicePluginConfig[x];
+            addSelector(String(F("p043_state")) + (x), 3, options, NULL, NULL, choice);
+          }
+          else addFormNumericBox(String(F("Temp")) + (x + 1), String(F("p043_state")) + (x), ExtraTaskSettings.TaskDevicePluginConfig[x],-100,100 );
```

Comment:
Again the "Temp".
It is using the same HTML id as the On/Off/"" selector, but it is still a bit unclear to me what its relation it has to the Clock Output.
You mentioned before it is indeed a temperature, so I would at least expect a unit of measure after the field (`addFormUnit`)
But I'm not yet convinced the label should have such a specific use case relation.

Reply:
Sorry, missed that. I will change it to "value", which is generic. Change is uploaded.